src/components/Modal/README.md

```diff
@@ -98,14 +98,14 @@ See [API](#api) for all available options.
 
 Modal is decomposed into the following components:
 
-- [Modal](#api)
-  - [ModalHeader](#modalheader_1)
-    - [ModalTitle](#modaltitle)
-    - [ModalCloseButton](#modalclosebutton)
-  - [ModalBody](#modalbody_1)
-    - [ModalContent](#modalcontent)
+- Modal
+  - ModalHeader
+    - ModalTitle
+    - ModalCloseButton
+  - ModalBody
+    - ModalContent
       (may be wrapped with [ScrollView](/components/ScrollView))
-  - [ModalFooter](#modalfooter_1)
+  - ModalFooter
```

Comment:
Looking at it, I'd like to bring the links back — at least `ModalHeader`, `ModalBody` and `ModalFooter` make sense as they have their own sections.

<img width="637" alt="obrazek" src="https://github.com/react-ui-org/react-ui/assets/5614085/cc3d7430-d95e-4041-af09-117f4acdbd80">

(Broken list indentation is another issue…)

Reply:
I came across another two occurrences of broken list indentation:

- On the `Card` component page, in the 'Composition' section
![image](https://github.com/react-ui-org/react-ui/assets/110407890/d1f4a701-ac8a-4461-b6de-55d79c2e6849)
- On the `Button` component page, in the 'Theming Priorities and Color Variants' section
![image](https://github.com/react-ui-org/react-ui/assets/110407890/8c01bb1f-c7bd-4161-9662-50c887045797)

Reply:
I created #486 to fix it.

Reply:
It does make sense to us and @mbohal to have links to subsections that are just below the defined links. What more, I have problems with linter which was the primary reason to those changes. I will try it again.